plugins/gitlab/icon_gitlab/util/util.py

```diff
@@ -0,0 +1,44 @@
+import json
+from datetime import datetime
+
+
+class Util:
+    @staticmethod
+    def clean_json(obj: dict) -> dict:
```

Comment:
Can we use the helper built into the SDK?

Reply:
No just because we need to convert some specific empty string fields to an empty dict which the helper doesn't provide.